Create lwm2m-agent-1022-1-1-updated-clients-sdk-to-version-containing-n-2-token-fix-561.md (#4503)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1022-1-1-updated-clients-sdk-to-version-containing-n-2-token-fix-561.md

---
date: ""

> title: Fixed Lwm2m operation execution delay

Reviewer: ```suggestion
title: Fixed LWM2M operation execution delay
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-5841
version: 1022.1.1
---

> Fixed an issue where Lwm2m agent failed to execute operations immediately after their creation.

Reviewer: ```suggestion
The LWM2M agent failed to execute operations immediately after their creation. This issue has been fixed.
```

> Fixed an issue where the agent would fail to refresh JWT tokens following periods of inactivity, preventing automatic recovery and requiring a manual service restart.

Reviewer: ```suggestion
The LWM2M agent previously failed to refresh JWT tokens following periods of inactivity, preventing automatic recovery and requiring a manual service restart. This issue has been fixed.
```
